server/publicapi/tests/items_service_test.py

```diff
@@ -213,24 +213,24 @@ def test_provides_request_object_to_superclass_if_not_given(self):
         instance.get(None, lookup)
 
         self.assertTrue(fake_super_get.called)
-        args, kwargs = fake_super_get.call_args
+        args, _kwargs = fake_super_get.call_args
         self.assertEqual(len(args), 2)
         self.assertIsInstance(args[0], ParsedRequest)
 
     def test_sets_query_filter_on_request_object_if_present(self):
         request = MagicMock()
-        request.args = MultiDict([('q', '{"language": "de"}')])
+        request.args = MultiDict([('filter', '{"language": "de"}')])
         lookup = {}
 
         instance = self._make_one()
         instance.get(request, lookup)
 
         self.assertTrue(fake_super_get.called)
-        args, kwargs = fake_super_get.call_args
+        args, _kwargs = fake_super_get.call_args
         self.assertGreater(len(args), 0)
 
-        query_filter = json.loads(args[0].where)
-        self.assertEqual(query_filter.get('language'), 'de')
+        query_filter = args[0].args['filters'][0]
+        self.assertEqual(query_filter.get('term').get('language'), 'de')
```

Comment:
Probably better to set the default value of the first `.get()` to an empty dictionary so that a missing `'term'` would result in a failed assertion instead of an error.